.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+  - package-ecosystem: "pip"
+    directory: "/llvm/docs"
+    schedule:
+      interval: "weekly"
```

Comment:
Maybe change this to at least monthly too? Weekly I think would create a lot of unnecessary churn.

Reply:
I agree with @boomanaiden154 I think starting with monthly would be best.

Reply:
Great! Sorry for the late reply, I'll update the PR right away